moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/plan_solutions.hpp

```diff
@@ -80,7 +80,7 @@ class PlanSolutions
  *  \param [in] solutions Vector of solutions to chose the shortest one from
  *  \return Shortest solution, trajectory_ of the returned MotionPlanResponse is a nullptr if no solution is found!
  */
-planning_interface::MotionPlanResponse
+static planning_interface::MotionPlanResponse
```

Comment:
Since the new `parallel_planning_callbacks.h` includes this header, the `static` is necessary to avoid multiple definitions.

Reply:
I think it should also make this function `inline` since the definition is provided with the header file.

Reply:
done